distribution/abi/Comptroller.json

```
@@ -2634,22 +2537,6 @@
     "stateMutability": "nonpayable",
     "type": "function",
     "signature": "0x6a56947e"
-  },
-  {
-    "constant": false,
-    "inputs": [
-      {
-        "internalType": "address",
-        "name": "contributor",
-        "type": "address"
-      }
-    ],
-    "name": "updateContributorRewards",
-    "outputs": [],
-    "payable": false,
-    "stateMutability": "nonpayable",
-    "type": "function",
-    "signature": "0x741b2525"
   }
 ]
 }
```

```suggestion
}

```
Nit: It's good to have newlines at the end of files for POSIX compliance.